spec/tapioca/dsl/compilers/protobuf_spec.rb

```diff
@@ -225,64 +212,16 @@ def resolve(symbol); end
               assert_equal(expected, rbi_for(:Cart))
             end
 
-            it "generates methods in RBI files for classes with Protobuf with enum field with defined type" do
-              add_ruby_file("protobuf.rb", <<~RUBY)
-                require 'google/protobuf/timestamp_pb'
-                require 'google/protobuf/wrappers_pb'
-
-                Google::Protobuf::DescriptorPool.generated_pool.build do
-                  add_file("cart.proto", :syntax => :proto3) do
-                    add_message "MyCart" do
-                      optional :value_type, :enum, 1, "MyCart.MYVALUETYPE"
-                    end
-                    add_enum "MyCart.MYVALUETYPE" do
```

Comment:
Similar to `T.untyped` test I don't think we can replicate this syntax, so I removed this test in favour of the previous one.